src/modules/logger/logger.cpp

```diff
@@ -1186,13 +1187,13 @@ int Logger::create_log_dir(tm *tt)
 
 bool Logger::file_exist(const char *filename)
 {
-	struct stat buffer;
+	struct stat buffer = {};
```

Comment:
Interesting, I didn't see that. Let's figure out how to approach this more efficiently.